usr/src/tools/env/release.sh

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/sh
+#
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+# Copyright 2017, Richard Lowe.
+
+#	release build only (no -D or -F)
```

Comment:
Perhaps this file should be called "non-debug", because my releases have both sets of bits in them.